.github/workflows/ci.yml

```diff
@@ -275,6 +276,28 @@ jobs:
             ./mvnw test ${MAVEN_TEST} -pl :trino-memsql -Dmemsql.license=${MEMSQL_LICENSE}
           fi
 
+  test-bigquery:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Though I guess it comes from `test-memsql`, `strategy.fail-fast` is for matrix build in my understanding.

Reply:
Yeah - will drop.